test/kubetest/log_utils.go

```diff
@@ -35,7 +35,7 @@ func makeLogsSnapshot(k8s *K8s, t *testing.T) {
 }
 
 func archiveLogs(testName string) {
-	file, err := os.Create(filepath.Join(logsDir(), testName, DefaultArchiveFile))
+	file, err := os.Create(filepath.Join(logsDir(), testName+".zip"))
```

Comment:
@denis-tingajkin  Please talk with @artem-belov since we need a cluster name here as well.